radixdlt/babylon-wallet-android
# Update pre-auth expiration models

## Description
This PR updates the pre-authorization expiration models based on the latest changes in Sargon.


## How to test

1. Send a pre-auth request with 10 seconds delay after signing
2. Slide to sign
3. Authenticate
4. Notice that the remaining time displayed for the pre-auth polling is the time set in the request minus the time used for signing (if it takes 4 seconds to sign, then the pre-auth status sheet will show 6 (10-4) remaining seconds)
